tsconfig.json

```diff
@@ -15,6 +15,7 @@
     "strictNullChecks": true,
     "skipLibCheck": true,
     "noEmitOnError": false,
-    "noImplicitOverride": true
+    "noImplicitOverride": true,
+    "resolveJsonModule": true
```

Comment:
This was added in order to be able to add strongly-typed translations in https://github.com/smilligan93/SR5-FoundryVTT/pull/842/files#diff-5de9f07f22b21c1e2d53b0427885d926e27ece69a4006c6fce1101c1c7b7381a - I'm happy pull this out if need be.